src/datasources/RelationshipsDatasource.ts

```diff
@@ -48,28 +50,30 @@ export const createRelationshipsNode = (relationshipsList, refId): Relationships
       },
     };
   }, nodeField);
-  const edges = new MutableDataFrame({
+  const edges: Partial<DataFrame> = {
     name: 'edges',
     fields: Object.keys(edgeField).map((key) => ({
       ...edgeField[key],
       name: key,
+      values: []
     })),
     meta: {
       preferredVisualisationType: 'nodeGraph',
     },
     refId,
-  });
-  const nodes = new MutableDataFrame({
+  };
+  const nodes: Partial<DataFrame> = {
```

Comment:
any reasons behind using `Partial` or was that how Grafana suggested to use the `Dataframe`?

Reply:
need a type that is not exactly Dataframe yet, but could be converted to Dataframe. 
We can define an interface ourselves, but this seemsed to be simpler